src/java/org/apache/cassandra/schema/SchemaKeyspace.java

```diff
@@ -567,9 +569,14 @@ private static void addTableParamsToRowBuilder(TableParams params, Row.SimpleBui
             builder.add("memtable", params.memtable.configurationKey());
 
         // As above, only add the allow_auto_snapshot column if the value is not default (true) and
-        // auto-snapshotting is enabled, to avoid RTE in pre-4.2 versioned node during upgrades
```

Comment:
this is leftover from the previous patch as we do not depend on what auto_snapshot is set to in cassandra.yaml so this piece of the comment is not relevant.